packages/expo-cli/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "bugsnag-expo-cli",
-  "version": "45.1.0",
+  "version": "45.1.23",
```

think this should be updated later as part of the release process
```suggestion
  "version": "45.1.0",
```

---

Sorry, yes, this was for testing and should be reverted, good catch!